pkg/filesystem/path/windows_format.go

```diff
@@ -41,6 +43,30 @@ func (p windowsParser) ParseScope(scopeWalker ScopeWalker) (next ComponentWalker
 			}
 			return next, windowsRelativeParser{stripWindowsSeparators(p.path[2:])}, nil
 		}
+
+		if (p.path[0] == '\\' && p.path[1] == '\\') || (p.path[0] == '/' && p.path[1] == '/') {
+			serverStart := 2
+			serverLen := strings.IndexAny(p.path[serverStart:], "\\/")
+			if serverLen != -1 {
```

Comment:
How should the path `\\` be interpreted? Right now we convert it to an absolute path.

Reply:
`\\` is definitely not a valid UNC path as a UNC path has to have both a server and share name. So I think keeping the current behaviour of parsing that as an absolute path seems reasonable to me.

Reply:
Gotcha! But we don't want that to throw an explicit error?

Reply:
Good point. I've updated it so that we are stricter and give errors on empty server or share names, and also give errors if it's just `//`.